packages/runtime/test-runtime-utils/src/mockLogger.ts

```diff
@@ -20,6 +20,12 @@ export class MockLogger extends TelemetryLogger implements ITelemetryLogger {
         this.events.push(event);
     }
 
+    sendErrorEvent(event: ITelemetryErrorEvent, error?: any) {
```

Comment:
i don't think we shoule be overriding the sendErrorEvent method that already exists on TelemetryLogger